types/jsqrcode/LICENSE

```diff
@@ -0,0 +1,21 @@
+    MIT License
```

Comment:
please do not include a separate licence files. all definition packages under DT are distributed under https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/LICENSE